tools/core_packages.txt

```diff
@@ -80,6 +85,7 @@ libtool
 libunbound
 libunistring
 libuv
+libversion
```

Comment:
Does this work with ruby scripts like version.rb?  Currently, version.rb complains that the ruby-libversion gem is not available when running commands such as `crew check -V cmake`.

Reply:
```
$ crew check -V cmake
<internal:/usr/local/lib64/ruby/site_ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require': cannot load such file -- ruby_libversion (LoadError)
Did you mean?  rubygems/version
	from <internal:/usr/local/lib64/ruby/site_ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
	from ../tools/version.rb:12:in `<main>'
```

Reply:
Libversion is the C library, `ruby-libversion` is the Ruby bindings to that library that version.rb requires. `crew install libversion && gem install ruby-libversion` will satisfy the dependencies of `version.rb`.

Reply:
Do we need a `ruby_libversion` gem package?

Reply:
Using `gem` on its own works fine, so I don't think there's a particularly strong need, but I don't mind either way.

Reply:
I only added libversion here because the container builds noted that it was being installed by default.

Reply:
It's already in core.rb, so...

Reply:
Yeah, I think we were supposed to keep those roughly in sync?

Reply:
oh, for some reason i thought we were talking about buildessential.rb, oops